<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-JUQ15IP" ignored="0" name="Functional-chrome" tests="3" failures="1" timestamp="2020-11-30T10:03:14 EST" time="9.854" errors="0">
  <testcase name="TC_fill_Student_RegistrationForm" time="0.471" classname="com.vndly.testCases.TC001_Student_RegistrationForm"/>
  <testcase name="validateText" time="0.041" classname="com.vndly.testCases.TC001_Student_RegistrationForm"/>
  <testcase name="unValidedText_Capture_A_Screenshot" time="0.026" classname="com.vndly.testCases.TC001_Student_RegistrationForm">
    <failure type="java.lang.AssertionError" message="expected [Thanks for submitting the formA] but found [Thanks for submitting the form]">
      <![CDATA[java.lang.AssertionError: expected [Thanks for submitting the formA] but found [Thanks for submitting the form]
at com.vndly.testCases.TC001_Student_RegistrationForm.unValidedText_Capture_A_Screenshot(TC001_Student_RegistrationForm.java:74)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- unValidedText_Capture_A_Screenshot -->
</testsuite> <!-- Functional-chrome -->
